pkg/master/controller.go

```diff
@@ -171,6 +173,7 @@ func (c *Controller) Start() {
 	c.runner.Start()
 }
 
+// Stop defines process of stopping the master
```

Comment:
Stop cleans up this API Servers endpoint reconciliation leases so another master can take over more quickly.

Reply:
done